### DISCUSSION TITLE
Does spacy_hunspell support multiple languages?

### DISCUSSION THREAD

**Discussion by mikeallen7** [explosion/spaCy#9954] [Language Support]
Hello. What is the best way to support spell check for multiple languages? 

We currently need to support 7 languages (English, French, German, Italian, Portuguese, Russian, and Spanish) and we plan to support more by the end of the year. 

Spacy_hunspell does what we want in English, but I am not finding much about whether it supports other languages. And, if yes, then what we would need to do to make it support these languages.

Or, is there a better option? If so, then could you kindly point us towards it? 

Thanks!

**Comment by DuyguA** [comment]
Hellos!

I checked the `spacy_hunspell` code for you, it loads an English dictionary:  https://github.com/tokestermw/spacy_hunspell/blob/31a7655244543aef7d67ed7921e81e0a677c4925/spacy_hunspell/__init__.py#L13

This package is based on Hunspell code and dictionaries. Hunspell offers their dictionaries for Linux and mac OS, here's where their 
English dictionary is located on my system (so the above code works):

```
duygu@duygu-Lenovo-ideapad-720-15IKB:~$ ls  /usr/share/hunspell
en_US.aff  en_US.dic
```

Hunspell offers dictionaries for the above langauges you listed, afaik. All you need to do find the precompiled dictionaries for your languages.  For example, for Ubuntu distro you can install these dictionaries via `apt` :  https://packages.ubuntu.com/search?keywords=hunspell  . If you look for packages for your distro's package manager I'm sure you can find :wink: 

Then replace `en_US`  with your desired language's code and ta tammm it should work :slightly_smiling_face:   

Please keep in my mind that we don't maintain this package, so if you need some changes in this package we cannot deliver it for you.

Cheers!
